docs/DeveloperGuide.md

```diff
@@ -921,36 +921,242 @@ testers are expected to do more *exploratory* testing.
-
-   1. Other incorrect delete commands to try: `sdel`, `sdel x`, `...` (where x is larger than the list size)<br>
+   3. Test case: `sdel 0`<br>
+      Expected: No stall is deleted. Error details shown in the status message. GUI does not update.
+   
+   4. Other incorrect stall delete commands to try: `sdel`, `sdel a`, `sdel -1`, `sdel x`(where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+### Deleting a review
+
+1. Deleting a review while all reviews are being shown
+
+    1. Prerequisites: List all reviews using the `rlist` command. Multiple reviews in the list.
+
+    2. Test case: `rdel 1`<br>
+       Expected: First review is deleted from the list. Details of the deleted review shown in the status message. GUI updates to remove the deleted review.
+
+    3. Test case: `rdel 0`<br>
+       Expected: No review is deleted. Error details shown in the status message. GUI does not update.
+
+    4. Other incorrect stall delete commands to try: `rdel`, `rdel a`, `rdel -1`, `rdel x`(where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Editing a stall
+
+1. Editing a stall while all stalls are being shown
+
+    1. Prerequisites: List all stalls using the `slist` command. Multiple stalls in the list.
+
+    2. Test case: `sedit 1 n/Alice Chicken Rice`<br>
+       Condition: There must not exist a stall with the same name and address as the edited stall's name and address<br>
+       Expected: The name of the first stall and its associated reviews edited. Details of the edited stall shown in the status message. GUI updates to show the edited details.
+
+    3. Test case: `sedit 1 a/Chicken Street 123`<br>
+       Condition: Another stall must not have the same name and address as the edited stall's name and address<br>
+       Expected: The address of the first stall and its associated reviews edited. Details of the edited stall shown in the status message. GUI updates to show the edited details.
+   
+    4. Test case: `sedit 1 n/Alice's Chicken Rice`<br>
+       Expected: No stall is edited. Error details shown in the status message. GUI does not update.
+
+    5. Other incorrect stall edit commands to try: `sedit`, `sedit 1`, `sedit x n/Alice Chicken Rice`, `sedit x a/Chicken Street 123`(where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Editing a review
+
+1. Editing a review while all reviews are being shown
+
+    1. Prerequisites: List all reviews using the `rlist` command. Multiple reviews in the list.
+
+    2. Test case: `redit 1 d/20/09/2022`<br>
+       Condition: There must not exist a review with the same details (date, content and tags) as the edited review's details<br>
+       Expected: The date of the first review edited. Details of the edited review shown in the status message. GUI updates to show the edited details.
```

Comment:
Maybe add that the review is moved to the back?

Reply:
Fixed.